### ISSUE TITLE
Error-Message: "Harter Fehler"

### ISSUE THREAD

**Issue by andit** [shadimanna/dhl-logistic-services-for-woocommerce#82]
Hi, 
is it possible to show in that cases more detailed information?
It happens sometime when some address fields are wrong, or if I switch to 2 Parcels, or...
But it costs always lots of time to find the real error...

**Comment by SeraphMcLight** [comment]
With "Harter Fehler" usually it's the address or if someone had put certain extra information in the shipping field such as "2. OG" etc. - At least this is the case with us. If you untick "Nur Leitcodierbare Adressen zulassen" you can work around the problem in that case BUT you will pay an extra fee with DHL for them to "correct" the address.

**Comment by andit** [comment]
yes, I know that there are in most cases some fields in the address wrong, but I would like to get a more detailed feedback from the program.
I would believe, that it parses the fields and can give at least a feedback which field is wrong, otherwise I have to check all fields and configurations to find the right combination...

**Comment by shadimanna** [comment]
Is this still an issue?

**Comment by andit** [comment]
sorry for the late reply,
yes I still think it was really helpful to get some more detailed information which fields were wrong.
